indexer/docs/README.md

```diff
@@ -0,0 +1,6 @@
+# @eth-optimism/indexer/docs
+
+The [vitepress](https://vitepress.dev/) based docs application for indexer
+
+IMPORTANT these are not official docs and will be moved to [real docs](https://community.optimism.io) if/when the indexer is released to GA
```

Comment:
oo good flag

Reply:
Yea no plans for these docs to exist.   It might look like I put effort into them but I didn't.  Vitepress is a cookie cutter template that pretty much does all the work for you of spinning up a docs site when you hand it the MD files. 

The first draft of the MD files was written by me passing our current JSON file into chat gpt.   This entire thing was incredibly low effort believe it or not

Reply:
You will see a vitepress site everytime I'm writing markdown files.   It's easiest to write and edit them in vitepress than any other way because you basically get a really fast dev server rendering the markdown for you